### DISCUSSION TITLE
rkhunter warn me after a "sudo firecfg"

### DISCUSSION THREAD

**Discussion by ghost** [netblue30/firejail#5234] [Q&A] [answered]
Hello,

Is that normal that all those binary are modified after a "sudo firecfg" ? Why ?

![Screenshot from 2022-07-05 14-05-28](https://user-images.githubusercontent.com/108735765/177323550-97969b4f-af49-48c7-a461-434b02a5f3f8.png)

Thx for your help :)

**Comment by rusty-snake** [comment, marked as answer]
> Is that normal that all those binary are modified after a "sudo firecfg" ? Why ?

The binaries itself are installed under `/usr/bin` and do not get modified. What happens here is that firecfg creates symlinks to firejail which the names of the binaries in a directory in `$PATH` with a higher order than `/usr/bin` (read it's documentation for more details) so you execute `firejail` instead of `ssh` if you type `ssh`. `firejail` then detects that it was started through a symlink named `ssh` and starts `ssh` in a sandbox.

TL;DR: Totally normal. 

You can check if you can configure rkhunter to skip `/usr/local/bin` to check the actual binaries.

**Reply by ghost** [reply]
Thank you for your answer, have a nice day.

**Reply by rusty-snake** [reply]
FYI #5236
